src/settings/DesktopSettingsViewer.js

```diff
@@ -226,6 +236,11 @@ export class DesktopSettingsViewer implements UpdatableSettingsViewer {
 		       })
 	}
 
+	appUpdate(): void {
```

Comment:
I would rename it into something like `onAppUpdateAvailable()`

Reply:
agreed